nekoyume/Assets/_Scripts/UI/Popup/ItemCountableAndPricePopup.cs

```diff
@@ -114,6 +118,20 @@ protected override void Awake()
                     _data.OnChangePrice.OnNext(price);
                 }).AddTo(_disposablesForAwake);
             }
+
+            reregisterButton.OnSubmitClick
+                .Subscribe(_ =>
+                {
+                    _data?.OnClickReregister.OnNext(_data);
+                    AudioController.PlayClick();
+                })
+                .AddTo(_disposablesForAwake);
+
+            notificationButton.OnClickAsObservable().Subscribe(_ =>
+            {
+                OneLinePopup.Push(MailType.System,
```

Comment:
No need play the click SFX here?

Reply:
Similar types of notifications did not include sfx. Please check again 🙏🏻 @Namyujeong